### PR TITLE
SIMSBIOHUB-500/140: Purge Old SIMS Code

### PR DESCRIPTION
**Pull request by KjartanE**
## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-500
- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-140

## Description of Changes

- Remove any unused code
- Clean out old snapshot tests

## Testing Notes

- {List any relevant testing considerations, necessary pre-reqs, and areas of the app to focus on. Specifically, include anything that will help the reviewers of this PR verify the code is functioning as expected.}

### CODE REVIEW

**Review by NickPhura — APPROVED** [review]

File: app/src/constants/misc.ts
Comment (curtisupshall) [inline app/src/constants/misc.ts:1]: Could these remaining two enums be moved to a more appropriate file?

**Review by al-rosenthal — APPROVED** [review]
👛

File: app/src/constants/misc.ts
Comment (curtisupshall) [reply app/src/constants/misc.ts:1]: Arguably not an important change, feel free to ignore

Comment (NickPhura) [reply app/src/constants/misc.ts:1]: Lets not worry about it for now

**Pull request merged**